### PR TITLE
Multiple improvements

### PR DESCRIPTION
**Pull request by scollazo**
- Reset MySQL ss db
- Improve FPR configuration. This change is not backwards-compatible, but the new dictionary allows for better configuration.
- Add ability to use customized logging settings depending on the environment/hostname
- Add ability to use customized config files for nginx
- Add ability to send logs to syslog

### CODE REVIEW

**Review by mamedin — COMMENTED** [review]
Nice improvement Santi :)

But I think it is a good idea to make configurable the variables for each AM service:
* datefmt
* format
* syslog address/port
* formatter
* facility
* level
 
The actual fixed values can be used perfectly as the default values.

**Comment by scollazo** [comment]
As there is a previous commit that allows to use custom .logging.json files, I think that datefmt, dateformat and formatter are not settings that we use to change.

On the other side, being able to configure the syslog address, facility and level seem good ideas, I'll add them to this pr

**Comment by scollazo** [comment]
An example rsyslog config (to be placed in /etc/rsyslog.d/archivematica.conf) could be as follows:

<pre>
#Create templates
$template SSFile,"/var/log/archivematica/%$YEAR%/%$MONTH%/%$DAY%/StorageService.log"
$template DashboardFile,"/var/log/archivematica/%$YEAR%/%$MONTH%/%$DAY%/Dashboard.log"
$template MCPCFile,"/var/log/archivematica/%$YEAR%/%$MONTH%/%$DAY%/MCPClient.log"
$template MCPSFile,"/var/log/archivematica/%$YEAR%/%$MONTH%/%$DAY%/MCPServer.log"

#Log each service on it's own file
local0.*  -?SSFile
local1.*  -?DashboardFile
local2.*  -?MCPCFile
local3.*  -?MCPSFile

#Stop processing , so we don't get duplicates in /var/log/messages
local0.* stop
local1.* stop
local2.* stop
local3.* stop
</pre>

File: templates/etc/archivematica/serverConfig.logging.json.j2 [hunk 1]
```diff
@@ -1,12 +1,26 @@
 {
   "disable_existing_loggers": false,
   "formatters": {
+{% if archivematica_src_syslog_enabled|bool %}
+    "syslog": {
+      "datefmt": "%Y-%m-%d %H:%M:%S %z",
+      "format": "{{ inventory_hostname }} MCPServer: [%(asctime)s] [%(process)d] [%(levelname)s]  %(name)s:%(module)s:%(funcName)s:%(lineno)d:  %(message)s"
+    },
```
Comment (mamedin) [inline templates/etc/archivematica/serverConfig.logging.json.j2:8]: Hi, there's a missing {% endif %} here

**Review by mamedin — CHANGES_REQUESTED** [review]

File: tasks/configure.yml [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+    - files:
+        - "templates/{{ inventory_hostname }}/rsyslog.conf.j2"
+        - "templates/rsyslog.conf.j2"
+        - "etc/rsyslog.d/archivematica.conf.j2"
+  tags: "amsrc-ss-osconf"
+  notify:
+    - Restart rsyslog
+  when: "archivematica_src_syslog_enabled|bool"
+
+
```
Comment (mamedin) [inline tasks/configure.yml:18]: On CentOS rsyslog is run as root user, but this service runs as syslog user on Ubuntu. To fix the permissions issue Ubuntu needs:
```
sudo chmod 2775 /var/log/archivematica
sudo chgrp syslog /var/log/archivematica
``` 
NOTE: `/var/log/archivematica`  is 2755 by default. The `/var/log` directory has 0775 permissions and `root:syslog` as owners.

**Review by mamedin — CHANGES_REQUESTED** [review]

File: tasks/configure.yml (hunk 2 above)
Comment (scollazo) [reply tasks/configure.yml:18]: The ```/var/log/archivematica``` permissions are set in https://github.com/artefactual-labs/ansible-archivematica-src/blob/qa/1.x/tasks/pipeline-osconf.yml#L76 , isn't that enough?

File: templates/etc/archivematica/serverConfig.logging.json.j2 (hunk 1 above)
Comment (scollazo) [reply templates/etc/archivematica/serverConfig.logging.json.j2:8]: Fixed

File: tasks/configure.yml (hunk 2 above)
Comment (scollazo) [reply tasks/configure.yml:18]: Fixed in 546ca6b

**Review by mamedin — APPROVED** [review]
NIce work

**Pull request merged**